fibonacci/cpp/code.cpp

```diff
@@ -0,0 +1,30 @@
+#include "stdio.h"
+#include "stdlib.h"
+#include "stdint.h"
+
+int32_t fibonacci(int32_t n) {
+    if (n == 0) return 0;
+    if (n == 1) return 1;
+    int32_t a = 0, b = 1, c;
+    for (int32_t i = 2; i <= n; i++) {
+        c = a + b;
+        a = b;
+        b = c;
+    }
+    return b;
+}
```

Comment:
This doesn't use the recursive algorithm, which AFAIUI, is the point of this benchmark.